pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -7443,7 +7443,7 @@ final: prev:
     meta.homepage = "https://github.com/ii14/neorepl.nvim/";
   };
 
-  neorg = buildVimPlugin {
+  neorg = buildNeovimPlugin {
```

Comment:
issue: this file is generated.

We should:

- delete neorg from here
- remove neorg it from vim-plugin-names
- add the following to the vim plugin overrides:

```nix
neorg = neovimUtils.buildNeovimPlugin { luaAttr = luaPackages.neorg };
```

(note that the `luaAttr` type has changed since #343564)

Reply:
Interesting, so is this the new approach to prevent duplicating definitions in both areas?

Reply:
It's existed for a while, but it's still under development (#335931).

The reason for using lua packages as the source of truth is because we don't have to declare dependencies manually.